.github/workflows/task-build-artifacts.yml

```diff
@@ -47,7 +47,33 @@ jobs:
         id: mode
         run: |
           [[ -z "${{ inputs.container }}" ]] && echo "mode=sudo" >> $GITHUB_OUTPUT || echo "mode=" >> $GITHUB_OUTPUT
+      - name: Check if node20 is Supported
+        id: node20 # TODO: Remove this when node20 is supported on all platforms, or when we drop support for theses platforms
+        run: |
+          for platform in ubuntu:bionic centos:7 amazonlinux:2; do
```

Comment:
centos7?

Reply:
It still belongs in the list, even if we don't run it on master.
It is required for 2.8 and 2.6